index.bs

```diff
@@ -1067,9 +1067,8 @@ associated.
 
     :   <dfn>requireResidentKey</dfn>
     ::  This member describes the [=[RPS]=]' requirements regarding availability of the [=Client-side-resident Credential 
-        Private Key=]. If the parameter is set to <a>true</a>, the authenticator MUST create a 
-        [=Client-side-resident Credential Private Key=] when creating a [=scoped credential=].
-
+        Private Key=]. If the parameter is set to true, the authenticator MUST create a 
```

Comment:
You should use "<code>&#x60;true&#x60;</code>" to typeset this, but again, the text you have isn't terrible.